toolchain/semantics/node_kind.h

```diff
@@ -17,6 +17,8 @@ struct NodeId {
 
   void Print(llvm::raw_ostream& out) const { out << "%" << id; }
 
+  operator int32_t() const { return id; }
```

Comment:
Is it important that this is implicit?

Reply:
No, this was leftover from intermediate changes -- restored the trunk version of this file.